### PR TITLE
Renaming T0 and V0 to comply with convention

### PR DESCRIPTION
**Pull request by rnevils**
All instances (variables, files, strings, etc) of "T0" and "V0" are to be changed to "FT0" and "FV0" for consistency, and to comply with the 3 letter naming convention of detectors.

### CODE REVIEW

**Comment by mslupeck** [comment]
@AllaMaevskaya could you also have a look?

File: Detectors/Base/test/testDCAFitter.cxx [hunk 1]
```diff
@@ -30,7 +30,7 @@ void checkResults(const DCAFitter& fitter, const std::array<float, 3> xyz)
     float dx = vtx.x - xyz[0], dy = vtx.y - xyz[1], dz = vtx.z - xyz[2];
     float dst = TMath::Sqrt(dx * dx + dy * dy + dz * dz);
 
-    const auto &trc0 = fitter.getTrack0(ic), &trc1 = fitter.getTrack1(ic); // track parameters at V0
+    const auto &trc0 = fitter.getTrack0(ic), &trc1 = fitter.getTrack1(ic); // track parameters at FV0
```
Comment (shahor02) [inline Detectors/Base/test/testDCAFitter.cxx]: The V0 here is for the secondary vertex, not the detector... Please undo the changes for this file.

File: Detectors/TPC/calibration/src/CalibPulser.cxx [hunk 2]
```diff
@@ -220,7 +220,7 @@ void CalibPulser::analyse()
 void CalibPulser::dumpToFile(const std::string filename)
 {
   auto f = std::unique_ptr<TFile>(TFile::Open(filename.c_str(), "recreate"));
-  f->WriteObject(&mT0,    "T0");
+  f->WriteObject(&mT0, "FT0");
```
Comment (shahor02) [inline Detectors/TPC/calibration/src/CalibPulser.cxx]: Here also, T0 refers to time0 of the TPC trigger, not detector, please change back.

File: EventVisualisation/DataConverter/include/EventVisualisationDataConverter/ConversionConstants.h [hunk 3]
```diff
@@ -20,24 +20,24 @@
...
 
-    KinkMother,
-    KinkDaughter,
+  KinkMother,
+  KinkDaughter,
 
-    V0NegativeDaughter,
-    V0PositiveDaughter,
-    V0Mother,
+  FV0NegativeDaughter,
```
Comment (shahor02) [inline EventVisualisation/DataConverter/include/EventVisualisationDataConverter/ConversionConstants.h]: Please revert this also, V0 here is for the secondary vertex.
